mountpoint-s3/src/cli.rs

```diff
@@ -54,7 +53,7 @@ Arguments:
     after_help = if cfg!(feature = "fstab") {FSTAB_DOCS} else {""},
 )]
 pub struct CliArgs {
-    #[clap(help = "Name of bucket to mount", value_parser = parse_bucket_name)]
+    #[clap(help = "Name of bucket to mount")]
     pub bucket_name: String,
```

Comment:
Should we update the help here? Do we want to encourage any adoption of S3 URIs or discourage it?

```suggestion
    #[clap(help = "Name of bucket, or an S3 URI, to mount")]
    pub bucket_name: String,
```